### PR TITLE
Add aiohttp_debugtoolbar to make debugging the web installer easier

### PR DESCRIPTION
**Pull request by cmaloney**
Enabled whenever `--web --verbose` is run.

### CODE REVIEW

**Comment by orsenthil** [comment]
Would  `--debug` be a better flag as that's the purpose of this hook?

File: dcos_installer/async_server.py [hunk 1]
```diff
@@ -360,6 +361,10 @@ def start(cli_options):
     make_default_config_if_needed('genconf/config.yaml')
     loop = asyncio.get_event_loop()
     app = build_app(loop)
+
+    if options.verbose:
```
Comment (orsenthil) [inline dcos_installer/async_server.py]: Instead of verbose, should it be a debug? 

(that's my only comment. everything else LGTM).

**Comment by cmaloney** [comment]
`--verbose` is pre-existing and already defined as `--debug` currently. Trying to be a minimal change here of making debugging the aiohttp app easier.

**Comment by orsenthil** [comment]
Okay. reasoning sounds good to me.

Ship-it.

**Comment by cmaloney** [comment]
Things which aren't passing all the tests shouldn't ever be given ship its...

**Comment by orsenthil** [comment]
Okay. I will review it again once you have gotten it right.

**Comment by cmaloney** [comment]
Closing for now, no time to work on it. Will re-work later. Will help when we rework the web apis

**Pull request closed without merging**